app/views/notifications_mailer/hsa_decision_client.text.haml

```diff
@@ -2,7 +2,7 @@ Hello #{@contact.full_name},
 \
 = "A #{Translation.translate('Housing Subsidy Administrator')} has made a decision on a housing opportunity for which the client below applied."
 \
-Decision: #{@match.hsa_accepts_client_decision.status_label}
+Decision: #{@match.eleven_hsa_accepts_client_decision.status_label}
```

Comment:
This looks like a mistake, since this is probably the original file for the provider only route